Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager
 
 # Copy the controller-manager into a thin image
 FROM ubuntu:latest
+RUN apt-get update --allow-releaseinfo-change && apt-get upgrade -y
```

Comment:
Why is `--allow-releaseinfo-change` required?

Reply:
https://manpages.debian.org/unstable/apt/apt-get.8.en.html#:~:text=%2D%2Dallow%2Dreleaseinfo%2Dchange,e.g%20a%20new%20major%20release.